backend/services/main-service/utils/packed_data/users.json

```diff
@@ -0,0 +1,152 @@
+[
+  {
+    "userId": 1,
+    "username": "user123",
+    "email": "user123@example.com",
+    "password": "hashedpassword1",
+    "dateOfBirth": "1990-05-21T00:00:00Z",
+    "name": "Alice Smith",
+    "country": "Canada",
+    "languages": ["English", "Spanish"],
+    "tags": ["traveler", "photographer"],
+    "boards": [],
```

Comment:
If it's not much of a hassle, would it be possible for you to add some of the boards and posts you've / you'll add? Thank you

Reply:
I now just have a sample testuser, with linked itineraries, posts, boards etc